src/fragments/lib/storage/flutter/configureaccess/50_customization.mdx

```diff
@@ -0,0 +1,84 @@
+## Customization
+
+### Customize Object Key Path
+
+You can customize your key path by defining a prefix resolver:
```

Comment:
@fjnoyp - can you please elaborate on the use case here? Why would a customer want to do this basically.

Reply:
I copied this from the Android and IOS template so if we change this we should probably change Android and iOS docs as well for consistency purposes.

The brevity here is because the previous section of storage explain how private/public/protected uploads are stored with certain prefixes. 

https://docs.amplify.aws/lib/storage/configureaccess/q/platform/android/#protected-access

It might be a bit redundant then to give motivation.  

I wrote a new prompt that elaborates more, but after reading through the entire article I agree with Android/iOS team approach and avoiding redundancy here. 

```
As mentioned above, configuring an access level in storage upload options will cause your files to be uploaded with a _public/_ _protected/_ or _private/_ prefix.

If you would like your items to be stored with a different prefix, you can customize your key path by defining a prefix resolver:
```

Reply:
yes, I like this a lot! Thank you, Kyle!

Reply:
Great done!